samples/led/main.c

```diff
@@ -1,22 +1,17 @@
+#include <hal/led.h>
 #include <hal/xbox.h>
-#include <xboxkrnl/xboxkrnl.h>
-
-#define SMC_REG_LEDMODE             0x07
-#define SMC_REG_LEDSEQ              0x08
 
 void main(void)
 {
-    // Set LED to be user controlled
-    HalWriteSMBusValue(0x20, SMC_REG_LEDMODE, FALSE, 0x01);
+  // Set front LEDs to shine green and then red.
+  // For information about timings and such, see
+  // https://www.XboxDevWiki.net/LED
+  XSetCustomLED(LED_OFF, LED_OFF, LED_OFF, LED_OFF);
 
-    // LED pattern is:
-    //   Red:   0xC = 1100 (ON,  ON,  OFF, OFF;   ON,  ON,  OFF, OFF;   ...)
-    //   Green: 0x0 = 0000 (OFF, OFF, OFF, OFF;   OFF, OFF, OFF, OFF;   ...)
-    // This means the red LED will blink slowly and the green LED is off
-    HalWriteSMBusValue(0x20, SMC_REG_LEDSEQ, FALSE, 0xC0);
+  // XResetLED() can be used to restore kernel control of the front LEDs
```

Comment:
The kernel is never really in control of it; I'd call it `system control` probably (the SMC handles it, but I think putting SMC here is too cryptic)